### ISSUE TITLE
Option to export schema only

### ISSUE THREAD

**Issue by fabioc-ms** [DaxStudio/DaxStudio#545]
I really like the Export Data advanced feature. However, it would be great to have an option NOT to download the data. 

Thanks,

Fabio

**Comment by dgosbell** [comment]
I assume this is for the SQL export option only. What is an example use case of when you would want to do this?

**Comment by fabioc-ms** [comment]
Yes, I mean the SQL export option. The scenario is when the cube is way too large. I tried the option and the second of 50 tables had 30 million lines. I gave up after two hours it had transferred only 10% of the lines of the second table... Maybe there should be three options:
- Normal full export
- Schema only
- Sample fist 100 (or N) rows of each table

**Comment by marcosqlbi** [comment]
Fabio, I'm curious too - why do you want to export the schema only of the first 100 rows?

**Comment by fabioc-ms** [comment]
Hi Marco.
In my setting, I need to consume data from cubes, sql servers, data lakes, etc, etc. and centralize in a staging sql database. Consuming data from SSAS into SQL can be a pain! So I am thinking of sampling the data from the structure of the cube in a sql database and then writing the DAX to bring in only the data that I need. Maybe I am wrong, but I hope this way I can modify the resulting tables and make my process somewhat less painful.

**Comment by marcosqlbi** [comment]
I'm not getting this step:
_So I am thinking of sampling the data from the structure of the cube in a sql database and then writing the DAX to bring in only the data that I need._
If you do that, you would see a sample of the data in a SQL table, but you cannot write a DAX query there.
More than the sample data, it seems you want to have a single catalog of all the data sources (table names and column names) in a SQL Server database. When you have that, you don't have any measure (which could be an important part of the semantic model) and you have only the content of data that has been imported from other data sources (maybe another SQL Server database!).
I'm curious to understand why you have access to the semantic model of Analysis Services or Power BI and not to the underlying data sources.

**Comment by fabioc-ms** [comment]
Hello Marco, and thanks for engaging in this conversation. I am a big DAX Studio fan, and I learned a lot by using it. 
No, I don't plan to write DAX against the SQL tables. I plan to run DAX against the semantic model to populate the data that I need later. It is a way to better understand the model and leverage the work. 
But, now you made me curious. What is the intended use case for the feature in its current version?

**Comment by marcosqlbi** [comment]
It was a request made by users who wanted to access the raw data in a Tabular model because they didn't have access to the original data source. We were surprised by the request, but it seems that are several people in that same condition.

**Comment by fabioc-ms** [comment]
Yes, this is common in my experience. Many teams maintain tabular models but don't share access to underlying data for many reasons, including enforcement of business or privacy rules, or just because their database is really messy and don't want others to see that.... :)

**Comment by marcosqlbi** [comment]
Not directly. If you just want the schema, the Tabular model is included in the VPAX file you can generate from "Export Metrics" in the Advanced ribbon. However, if you want to create a relational database that corresponds to the tables in the Tabular model, this is not available today.
